src/main/java/pl/net/rogala/eventy/service/UserService.java

```diff
@@ -32,6 +34,18 @@ public void setDefaultRole(User user) {
         user.addRole(role);
     }
 
+    /**
+     * Adding new role (organizer) to User's Set of Roles
+     *
+     * @param user
+     */
+//
```

Comment:
// ?? :-)

Reply:
to jest User, któremu trzeba dopisać nową rolę (organizator)

Reply:
Chodziło mi o 42 linię i komentarz `//` ;-)

Reply:
done :)